src/drivers/src/stm32f4xx.c

```diff
@@ -0,0 +1,87 @@
+#include "can_raw.h"
+
+#include "stm32f4xx_hal.h"
+#include <stdint.h>
+
+#define USING_CAN2
```

Comment:
This define should again be made in user code before the include, since this is not just for VCU.
Also have you tested `USING_CAN1` and `USING_CAN3`?